src/node/blockstorage.cpp

```diff
@@ -390,10 +390,10 @@ bool BlockManager::IsBlockPruned(const CBlockIndex* pblockindex)
     return (m_have_pruned && !(pblockindex->nStatus & BLOCK_HAVE_DATA) && pblockindex->nTx > 0);
 }
 
-const CBlockIndex* GetFirstStoredBlock(const CBlockIndex* start_block) {
+const CBlockIndex* BlockManager::GetFirstStoredBlock(const CBlockIndex& start_block)
```

Comment:
In theory `start_block` could be marked LIFETIMEBOUND.

Reply:
> In theory `start_block` could be marked LIFETIMEBOUND.

Ah, yes; done in #25060.